pkg/apis/apps/v1alpha1/subscriptionstatus_types.go

```diff
@@ -19,128 +19,163 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// SubscriptionStatus provides detailed status for all the resources that are deployed by the application in a cluster.
 // +kubebuilder:object:root=true
```

Comment:
@philipwu08  I saw the subscriptionStatus structure was updated, It seems the subscriptionStatus CRD was missing to update